bodhi/server/scripts/approve_testing.py

```diff
@@ -103,6 +103,12 @@ def main(argv=sys.argv):
                 if update.autotime and update.days_in_testing >= update.stable_days:
                     print(f"Automatically marking {update.alias} as stable")
                     update.set_request(db=db, action=UpdateRequest.stable, username="bodhi")
+                    # For updates that do not stay in testing at all, mark the update as stable
+                    # Do note that if we ever configure a release which is composed by bodhi with
+                    # mandatory_days_in_testing = 0, its updates will end up being marked as stable
+                    # before the compose. This may lead to some unexpected behavior.
+                    if update.release.mandatory_days_in_testing == 0:
+                        update.status = UpdateStatus.stable
```

Comment:
I am not 100% sure but we might need to set the update.request to None also.

Reply:
Do we want to do it now or try this way and adjust later if needed?

Reply:
Yes I think we need to do it looking at what the composer does for normal updates. https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/consumers/composer.py#L664

Reply:
What about the `update.locked` flag?

Reply:
indeed, :+1:

Reply:
> What about the `update.locked` flag?

I think this is not needed ( reading the comment here https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L1646 )

Reply:
> What about the `update.locked` flag?

We don't lock them in our process, they are locked by the compose process itself so packagers do not update them while the compose is running :)